mongomock/aggregate.py

```diff
@@ -1581,6 +1607,9 @@ def _handle_out_stage(in_collection, database, options):
 
 
 def _handle_count_stage(in_collection, database, options):
+    print(f"{in_collection=}")
+    print(f"{database=}")
+    print(f"{options=}")
```

Comment:
I guess this was for debugging purposes?

Reply:
It was, I should have removed this prior to submitting the PR. I was trying to understand how the library works. Removing this.